README.md

```diff
@@ -684,7 +684,7 @@ Developers in the project may not follow consistent code security practices, lea
 **Otherwise:** Source control for even private repositories, can mistakenly be made public, at which point all secret has been exposed outside. Access to source control for an external party will inadvertently provide access to related systems (database, apis, etc).
```

Comment:
@js-kyle We should encourage devops way of thinking - source control is for developers, the production secrets must be visible to the people who operate the production only. Even if made private, a source control is not the appropriate system to store highly sensitive data